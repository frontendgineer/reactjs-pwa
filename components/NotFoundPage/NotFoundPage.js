import React from 'react'
import {Route} from 'react-router-dom'

const NotFoundPage = () => (
         <h1>Not Found Page</h1>
)

export default NotFoundPage
