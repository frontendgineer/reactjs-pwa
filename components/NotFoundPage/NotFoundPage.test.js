import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFoundPage from './NotFoundPage'

test('NotFoundPage component should render as expected', () => {
    const component = shallow(<NotFoundPage />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
})
