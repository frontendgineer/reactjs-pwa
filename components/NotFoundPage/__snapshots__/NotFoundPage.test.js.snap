// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component should render as expected 1`] = `
<h1>
  Not Found Page
</h1>
`;
